### PR TITLE
Output ng generate

### PR DESCRIPTION
**Pull request by pkozlowski-opensource**

### CODE REVIEW

File: packages/core/schematics/rollup.config.js [hunk 1]
```diff
@@ -25,9 +25,7 @@ const stripBannerPlugin = {
 
     return {
       code: magicString.toString(),
-      map: magicString.generateMap({
-        hires: true,
-      }),
```
Comment (devversion) [inline packages/core/schematics/rollup.config.js:30]: Is that intentional?

**Review by devversion — APPROVED** [review]

File: packages/core/schematics/rollup.config.js (hunk 1 above)
Comment (alan-agius4) [reply packages/core/schematics/rollup.config.js:30]: Yes, we do not emit sourcemaps here.

Comment (pkozlowski-opensource) [reply packages/core/schematics/rollup.config.js:30]: Semi-intentional: testing for OoO errors from rollup

**Review by alan-agius4 — APPROVED** [review]
LGTM

**Comment by AndrewKushnir** [comment]
This PR was merged into the repository by commit 35954455e2eeda54efaead02b1fdda2381f99898.

The changes were merged into the following branches: main

**Pull request closed without merging**